packages/web/src/pages/artist-dashboard-page/components/USDCCard.tsx

```diff
@@ -33,7 +35,10 @@ const messages = {
   withdrawalHistory: 'Withdrawal History'
 }
 
+const MediumWithdrawIcon = () => <Icon icon={IconWithdraw} size='medium' />
```

Comment:
nit: I personally would prefer this inlined

Reply:
Worth discussion. I generally avoid redefining function props on every render if I can help it, unless I'm really sure it won't result in anything below here re-rendering.

I assume that if you change the value of this prop each time, the button under the hood is going to be mounting a new instance of it every time as well and that might result in React doing updates we don't really need to do.

But I could probably test this first and make sure it's not smart enough to figure out that the resulting tree is the same and not do the extra work.

Reply:
Oh does it have to be a function prop? I didn't realize that - thought we could pass in the Icon component directly. Nevermind then, carry on!

Reply:
You are correct, normally we _should_ just pass in the component directly and let the button render an instance of it with the appropriate styling.
And as it turns out, the button _does_ do the right thing in this case. I had thought this was originally an override to fix the icon being the wrong size due to the source icon file not being the right scale.
But it looks correct to me if I just pass the icon in directly. So I will remove this intermediate component.

Thanks for asking questions!